searching/LinearSearch.c

```diff
@@ -24,4 +25,9 @@ void main(){
 		printf("Value %d is in the array.\n", v);
 	else
 		printf("Value %d is not in the array.\n", v);
+	printf("Want to search again press 1");
+	scanf("%d",y);
+	if(y!=1)
+	exit;
```

Comment:
**exit(0)** is a function. You missed the parenthesis.
You need to _include_ **stdlib.h** for __exit(0)__ to work.
or you could simply use **break** instead of exit